cmd/csi-node-driver-registrar/node_register.go

```diff
@@ -63,6 +65,19 @@ func nodeRegister(csiDriverName, httpEndpoint string) {
 	// Registers kubelet plugin watcher api.
 	registerapi.RegisterRegistrationServer(grpcServer, registrar)
 
+	// Sometimes on windows after registration with the kubelet plugin we don't
```

Comment:
Can you open up a separate bug to investigate why we don't get the callback sometimes?

Reply:
we have #143, I marked this PR as a workaround instead (it's no longer going to fix #143)